src/perf/tracepoint/exit_reader.cpp

```diff
@@ -38,12 +38,18 @@ static const EventFormat& get_sched_process_exit_event()
     return evt;
 }
 
-ExitReader::ExitReader(int cpu, trace::Trace& trace)
-: Reader(cpu, get_sched_process_exit_event().id(), config().mmap_pages), trace_(trace),
+ExitReader::ExitReader(int cpu, trace::Trace& trace) try
```

Comment:
If the try-method was intentional, please make a comment, as this syntax is pretty unusual.